python/tvm/driver/tvmc/common.py

```diff
@@ -295,10 +299,20 @@ def target_from_cli(target):
             raise TVMCException(f"Error parsing target string '{target}'.\nThe error was: {ex}")
 
         validate_targets(parsed_targets)
-        target = parsed_targets[-1]["raw"]
-        extra_targets = parsed_targets[:-1] if len(parsed_targets) > 1 else []
+        tvm_targets = [t for t in parsed_targets if t["is_tvm_target"]]
+
+        # Validated target strings have 1 or 2 tvm targets, otherwise
+        # `validate_targets` above will fail.
+        if len(tvm_targets) == 1:
+            target = tvm_targets[0]["raw"]
+            target_host = None
+        elif len(tvm_targets) == 2:
```

Comment:
I know there already has a checker before invoking this function, but it would be better to use assert to make our purpose clear.
```suggestion
        else:
          assert len(tvm_targets) == 2
```

Reply:
Yeah, makes sense.